### PR TITLE
Parse body of PATCH requests

### PR DESCRIPTION
**Pull request by calve**
Title say it all; allow to parse body of PATCH requests

should fix https://github.com/nbs-system/naxsi/issues/358

### CODE REVIEW

**Comment by buixor** [comment]
Thanks for the MR, giving it a look :)

**Comment by buixor** [comment]
@calve would you mind adding some unit tests ? In the meanwhile I'm going to check if there are some pitfalls we might take care of when dealing with `PATCH` method.

File: README.md [hunk 1]
```diff
@@ -37,9 +37,8 @@ is to add required ACCEPT rules for the target website to work properly.
 
 Contrary to most Web Application Firewalls, Naxsi doesn't rely on a
 signature base like an antivirus, and thus cannot be circumvented by an
-"unknown" attack pattern. Another main difference between Naxsi and other
-WAFs, Naxsi filters only GET and POST requests,
```
Comment (jvoisin) [inline README.md]: Only `GET`, `POST` and `PATCH`. We're not handling `HEAD`, `DELETE`, …
Shouldn't we keep this part?

**Comment by calve** [comment]
Thanks for the merge. Sorry I did not took time to add proper tests. On the other hand, there is no test for PUT requests aswell. Shall we copy/paste the tests for POST or factorise them ?

**Comment by jvoisin** [comment]
Yup, this is the way to go :)

**Pull request merged**